R/summaries.R

```diff
@@ -334,8 +334,7 @@ setMethod("countDominantFeatures", signature = c(x = "SummarizedExperiment"),
     tallied_data <- data %>%
         tally() %>%
         mutate(
-            rel.freq = round(100 * n / sum(n), 1),
-            rel.freq.pct = paste0(round(100 * n / sum(n), 0), "%")
+            rel.freq = round(n / sum(n), 3)
```

Comment:
If this is part of the function then provide the rounding digits as a function arguments (setting default to 3 is OK) so that users can ultimately choose as they like.